upstairs/src/client.rs

```diff
@@ -2252,7 +2279,8 @@ impl DownstairsClient {
     }
 
     pub(crate) fn total_live_work(&self) -> usize {
-        (self.io_state_count.new + self.io_state_count.in_progress) as usize
+        (self.io_state_job_count.new + self.io_state_job_count.in_progress)
```

Comment:
I don't know how this snuck through, but the `io_state_job_count.new` has no `IOState` equivalent any longer.  Since your in the neighborhood can you just remove the new field from `ClientIOStateCount`?

Reply:
Good catch, removed in 663ae0d00acb

This will change the DTrace probe format, but it looks like no one is using `ok.ds_io_count.new`